spec/seeder.test.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose');
+const db = require('../database/index.js');
+const seed = require('../database/seeder.js').seed;
+
+describe('Database seeder', () => {
+  let reviews;
+
+  beforeAll(async () => {
+    await seed();
+    await db.Review.find().then(results => reviews = results);
+  });
+
+  test('should populate reviews collection with 100 documents', () => {
+    expect(reviews.length).toBe(100);
+  });
+
+test("should return valid document by Film Name", async () => {
+  ( { "name.last": "Hopper" } )
+    const result = await db.Review.find({"filmname" : reviews[0].filmname});

```

Comment:
This test appears to be testing the content rather than the functionality of the database - it will likely fail if the content of the database ever changes.